<ui version="4.0" >
 <class>medium</class>
 <widget class="QDialog" name="medium" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>490</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <pointsize>9</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="8" >
    <widget class="QLabel" name="label_15" >
     <property name="font" >
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text" >
      <string> Label- und Karten Auswahl</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="8" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label" >
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>  Parameter  Print-Medium      </string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLabel" name="label_24" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>155</red>
           <green>62</green>
           <blue>54</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>155</red>
           <green>62</green>
           <blue>54</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text" >
      <string>Aktiv : </string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QLineEdit" name="TraegerW" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>155</red>
           <green>62</green>
           <blue>54</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>155</red>
           <green>62</green>
           <blue>54</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="QLabel" name="label_16" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>155</red>
           <green>62</green>
           <blue>54</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>155</red>
           <green>62</green>
           <blue>54</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" >
    <widget class="QLineEdit" name="TraegerTypeW" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>155</red>
           <green>62</green>
           <blue>54</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>244</green>
           <blue>211</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>155</red>
           <green>62</green>
           <blue>54</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>244</green>
           <blue>211</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
    </widget>
   </item>
   <item row="2" column="7" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="8" row="3" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="RowsW" >
      <property name="geometry" >
       <rect>
        <x>24</x>
        <y>42</y>
        <width>55</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="ColsW" >
      <property name="geometry" >
       <rect>
        <x>108</x>
        <y>42</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_3" >
      <property name="geometry" >
       <rect>
        <x>30</x>
        <y>18</y>
        <width>49</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text" >
       <string>Reihen</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4" >
      <property name="geometry" >
       <rect>
        <x>114</x>
        <y>18</y>
        <width>55</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text" >
       <string>Spalten</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5" >
      <property name="geometry" >
       <rect>
        <x>6</x>
        <y>78</y>
        <width>193</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Labelabstand vom Tr√§gerrand</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6" >
      <property name="geometry" >
       <rect>
        <x>24</x>
        <y>102</y>
        <width>73</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text" >
       <string>X-Richtung</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7" >
      <property name="geometry" >
       <rect>
        <x>108</x>
        <y>102</y>
        <width>73</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text" >
       <string>Y-Richtung</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="RimxW" >
      <property name="geometry" >
       <rect>
        <x>25</x>
        <y>120</y>
        <width>55</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="RimyW" >
      <property name="geometry" >
       <rect>
        <x>110</x>
        <y>120</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_8" >
      <property name="geometry" >
       <rect>
        <x>17</x>
        <y>156</y>
        <width>157</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text" >
       <string>Gr√∂√üe:  Label oder Karte</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9" >
      <property name="geometry" >
       <rect>
        <x>30</x>
        <y>174</y>
        <width>43</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text" >
       <string>Breite</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10" >
      <property name="geometry" >
       <rect>
        <x>108</x>
        <y>174</y>
        <width>67</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text" >
       <string>H√∂he</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="BreiteW" >
      <property name="geometry" >
       <rect>
        <x>24</x>
        <y>193</y>
        <width>55</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="HoeheW" >
      <property name="geometry" >
       <rect>
        <x>108</x>
        <y>193</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_11" >
      <property name="geometry" >
       <rect>
        <x>12</x>
        <y>228</y>
        <width>181</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string>Labelabstand untereinander</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12" >
      <property name="geometry" >
       <rect>
        <x>12</x>
        <y>252</y>
        <width>79</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text" >
       <string>zum unteren</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13" >
      <property name="geometry" >
       <rect>
        <x>108</x>
        <y>252</y>
        <width>79</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text" >
       <string>zum rechten</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="DifyW" >
      <property name="geometry" >
       <rect>
        <x>24</x>
        <y>271</y>
        <width>55</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="DifxW" >
      <property name="geometry" >
       <rect>
        <x>108</x>
        <y>271</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_14" >
      <property name="geometry" >
       <rect>
        <x>6</x>
        <y>306</y>
        <width>187</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string> Textabstand vom Labelrand</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PrTextW" >
      <property name="geometry" >
       <rect>
        <x>36</x>
        <y>325</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font" >
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_2" >
      <property name="geometry" >
       <rect>
        <x>6</x>
        <y>360</y>
        <width>187</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="1" colspan="7" >
    <widget class="QLabel" name="label_23" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="7" >
    <widget class="QLabel" name="label_17" >
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text" >
      <string>Auswahl vorgefertigte Datentr√§ger</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="7" >
    <widget class="QTreeWidget" name="mediumList" >
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Datentr√§ger</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Aktiv</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Breite</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>H√∂he</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Spalten</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Reihen</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Rand_x</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Rand_y</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>L.Abstand_x</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>L.Abstand_y</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Text.Pos</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Drucker</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Zuf.</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Rahmen</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Just.</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="6" column="1" colspan="7" >
    <widget class="QLabel" name="label_22" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>10</red>
           <green>92</green>
           <blue>26</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>10</red>
           <green>92</green>
           <blue>26</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>Clicke in die Liste um den Datentr√§ger zu tauschen oder zu bearbeiten - dann speichern   </string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="4" colspan="2" >
    <widget class="QLabel" name="label_25" >
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>Papierzuf√ºhrung  </string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="6" >
    <widget class="QLineEdit" name="ZufW" >
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </item>
   <item row="7" column="7" >
    <widget class="QLabel" name="label_26" >
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>Grad</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="5" >
    <widget class="QLabel" name="label_18" >
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>Labelrahmen mit Linienst√§rcke  </string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="6" >
    <widget class="QLineEdit" name="FrameW" >
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </item>
   <item row="8" column="7" >
    <widget class="QLabel" name="label_19" >
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>zeichnen      </string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="5" >
    <widget class="QLabel" name="label_20" >
     <property name="font" >
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>[ TEST - Nur Labelrahmen zeichnen ]  </string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="6" >
    <widget class="QLineEdit" name="JustW" >
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
   </item>
   <item row="9" column="7" >
    <widget class="QLabel" name="label_21" >
     <property name="font" >
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text" >
      <string>ohne Text       </string>
     </property>
    </widget>
   </item>
   <item row="10" column="3" colspan="2" >
    <widget class="QPushButton" name="ButtonUpdate" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>206</red>
           <green>238</green>
           <blue>208</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>206</red>
           <green>238</green>
           <blue>208</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>206</red>
           <green>238</green>
           <blue>208</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text" >
      <string>Speichern</string>
     </property>
    </widget>
   </item>
   <item row="10" column="5" colspan="3" >
    <widget class="QPushButton" name="ButtonEnde" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>225</red>
           <green>225</green>
           <blue>225</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>225</red>
           <green>225</green>
           <blue>225</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>225</red>
           <green>225</green>
           <blue>225</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text" >
      <string>Zur√ºck</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
